Emby.Naming/ExternalFiles/ExternalPathParser.cs

```diff
@@ -1,3 +1,5 @@
+#pragma warning disable CA1307
```

Comment:
Is there a reason you disable this? If not, making it compliant is fairly simple (just check out any other comparison in the code base)

Reply:
I just followed what was done in the existing code:
https://github.com/jellyfin/jellyfin/blob/38d0367c428fdce7e42c8e70da11327ecbba2c6c/Jellyfin.Server.Implementations/Item/BaseItemRepository.cs#L446

But apparently the code base is not consistent in its behavior of using `Contains`, and I will implement your suggestion instead.

Reply:
Done.

Reply:
It's disabled only for EFCore interactions